### ISSUE TITLE
EdgeReinforcement on edges defined by GridRef bounds are not supported in schema 3.0.0/3.0.1

### ISSUE THREAD

**Issue by nox-cadmatic** [OCXStandard/OCX_Schema#156]
It is impossible to properly define an EdgeReinforcement on an edge whose geometry is defined based on a topological bound that uses a GridRef. This is because an EdgeReinforcement requires at least one EdgeCurveRef which has its "ocx:refType" attribute limited to the "ocx:FreeEdgeCurve3D" type.
Situations where an edge reinforcement (faceplate) is added to a plate edge which is defined by a grid reference are common from my experience, so this is a problematic omission.
My suggested fix for this issue would be to allow either one more GridRef objects to be added to the ocx:EdgeReinforcement (instead of or in addition to EdgeCurveRef) or to extend the EdgeCurveRef to allow for references to ocx:ReferencePlane.

**Comment by ocastrup** [comment]
@nox-cadmatic Yes, this is a limitation today. The only way to overcome this today is to create a new TraceLine object duplicating the curve created by the limiting operation of the GridRef, see below. This is probably not a good idea so we should extend the definition of the EdgeCurveRef to point to the curve (it cannot point to a RefPlane).


![image](https://github.com/user-attachments/assets/853ab7f2-634c-4bb6-b7d0-583eeed253b6)

**Comment by ocastrup** [comment]
@nox-cadmatic @CarstenZerbst This case can be supported similarily to references to a hole by adding a choice of a GridRef:
![Image](https://github.com/user-attachments/assets/cba26425-6b03-4302-95e0-a50a6f702ea8)
